iocore/net/SSLConfig.cc

```diff
@@ -143,7 +143,7 @@ set_paths_helper(const char *path, const char *filename, char **final_path, char
 {
   if (final_path) {
     if (path && path[0] != '/') {
-      *final_path = ats_stringdup(RecConfigReadPrefixPath(nullptr, path));
+      *final_path = ats_stringdup(Layout::get()->relative_to(Layout::get()->prefix, path));
```

Comment:
Do you still need the `ats_strdup`? Doesn't `Layout::get()` do that?

Reply:
`Layout::get()->relative_to` returns a std::string

Reply:
Blech. Probably too much work to change `final_path` to be `std::string` based. Sigh.